README.md

```diff
@@ -46,6 +49,30 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | External DNS Service - ExternalDNS | ❌ | ✅ |
 *\* The embedded Harbor Registry feature is still available and supported on vSphere 7 and onwards.*
 
+## Consumption Interface
+
+Provides the Local Consumption Interface (LCI) for Namespaces within vSphere Client. This also includes the Single Sign On (SSO) component required by the Cloud Consumption Interface (CCI) in Aria Automation within VMware Cloud Foundation.
+
+The minimum required version for using this interface is 8.0U3.
+
+### Consumption Interface Versions
+
+Installation instructions can be found [here in VMware documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+NOTE: There are occasions where the plugin fails to load on the initial attempt. To check whether the plugin was loaded correctly, navigate to the vsphere Menu and click through to Administration -> Client Plugins. If you see a "Plug-in configuration with Reverse Proxy failed." message in the Status column for the "Namespace UI" plugin, you will need to reinstall.
+
+Download latest version:
+- [Consumption Interface v1.0.0](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/cci-supervisor-service/v1.0.0/cci-supervisor-service.yml)
+
+- [Release notes](./consumption-interface/Release_Notes_1_0_0.md)
+
+### OSS information
+
+[LCI OSS](https://github.com/vsphere-tmm/vsphere-with-tanzu-dev-center/tree/main/consumption-interface/ConsumptionInterface_LocalConsumptionInterface_1_0_0.zip)
```

Comment:
This link doesn't look right - is it supposed to reference the `.zip` file in this PR? If so, you can use [relative links](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files) to simplify that path. For example: `[LCI OSS](consumption-interface/ConsumptionInterface_LocalConsumptionInterface_1_0_0.zip)`

Reply:
Ah, wasn't sure that would work for download. Wiill adjust.